samples/interactive-tutorials/setup/delete-events-bigquery-table.js

```diff
@@ -0,0 +1,31 @@
+// Copyright 2022 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+async function main() {
+  const utils = require('./setup-cleanup');
```

Comment:
I'm probably just out of context here, so take this comment with a grain of salt, but I generally don't feel like the use of `utils` is beneficial in samples. If someone is really trying to follow this sample, it's harder to read when the code is splintered like this. Of course there are good reasons for using `utils`, but just my thoughts from the perspective of someone trying to use this sample (i.e., they can't just "copy and paste")

Reply:
Hi @sofisl, let me describe a bit the purpose of these code. It is Not the regular code samples which are in this repository. 
The task is to design the interactive tutorials which user will be able to run in CloudShell.
In the tutorials we ask user to set a GC project, and create a user account, and then user should open and run in the CloudShell the code samples. The tutorials are really granular, if it is about product updating we want show user only code related to using this method, and to avoid to overload user with extra code. so for theis particular example, we want to hide the methods for product creation and deletion in utils and present only the the update product method.

About using the env variables. We must use them for the bucket names (we use GCS buckets as an importing source) but we will make the Project_ID to be read from the gcloud config.